src/html-reporter/html.js

```diff
@@ -1027,6 +1033,34 @@ export function escapeText( s ) {
 		}
 	} );
 
+	QUnit.on( "error", ( error ) => {
+		stats.failedTests++;
+
+		const testItem = appendTest( "global failure" );
+		if ( !testItem ) {
+
+			// HTML Reporter is probably disabled or not yet initialized.
+			console.warn( "global failure" );
+			console.warn( error );
```

Comment:
warn vs error? I'm good with either, but maybe I was surprised to find this wasn't `console.error`; curious if that was intentional

Reply:
Yeah, it's an awkward fit, though note that we did also use `warn()` in onUncaughtException previously, for the same purpose (see left/red part of the diff).

This was a compromise to keep the diff minimal because we don't actually interface yet with `console.error()` at all. (Though I agree that we should!) It's be a bit more work to add it to our cross-realm interface and then adopt it in the places where it makes sense.